elks/arch/i86/lib/bios-none.S

```diff
@@ -0,0 +1,13 @@
+/*
+ *	NO BIOS
+ */
+
+#include <linuxmt/config.h>
+
+	.arch	i8086, nojumps
+	.code16
+	.text
+	.global	call_bios
+
+call_bios:
+	ret
```

Comment:
Do we have to have this file and the null call_bios function? My concern is other places in the kernel calling this function with SOLO/86 configured and not knowing the function isn't implemented. I can't tell exactly where this may be required to be linked in without compiling this PR prior to commit, so I thought I'd ask. 

I would hope that without the BIOSHD driver compiled in there would be no or very few calls to call_bios, but I'm guessing that's not the case. I would agree that it's probably not a great idea to ifdef out all the calls elsewhere, but we should probably discuss where these are just to make sure this makes sense.